package.json

```diff
@@ -50,5 +50,8 @@
   "mocha": {
     "timeout": 11000,
     "reporter": "spec"
+  },
+  "dependencies": {
+    "win-node-env": "^0.6.1"
```

Comment:
Should this have been installed gloabally to your node env not added to the project?